drivers/public_reports/app/models/public_reports/setting.rb

```diff
@@ -9,6 +9,16 @@ class Setting < GrdaWarehouseBase
     attr_encrypted :s3_access_key_id, key: ENV['ENCRYPTION_KEY'][0..31]
     attr_encrypted :s3_secret, key: ENV['ENCRYPTION_KEY'][0..31]
 
+    def self.available_map_types
+      types = {
+        coc: 'Continuum of Care',
+        # county: 'County',
```

Comment:
Should this be enabled?

Reply:
Commented out until we know how to calculate county for a project.